extlib/millhone/docs/subcommands/ingest.md

```diff
@@ -0,0 +1,35 @@
+# `ingest`
+
+This subcommand adds an open-source library to the FOSSA knowledge base.
+For more information on possible options, run `millhone ingest --help`.
+
+# Internal Users
```

Comment:
I'd suggest calling this `FOSSA Employees` to make it even more clear that customers cannot use this part of the functionality since `Internal` has a meaning which is relative to the person reading the docs.

Reply:
Addressed this and other doc comments here, on #1287: https://github.com/fossas/fossa-cli/pull/1287/commits/ffb0b29794ce8199c501c8a8fa1de3865afd1cd3